src/bacnet/basic/object/bi.c

```diff
@@ -615,3 +861,417 @@ bool Binary_Input_Write_Property(BACNET_WRITE_PROPERTY_DATA *wp_data)
 
     return status;
 }
+
+/**
+ * For a given object instance-number, gets the event-state property value
+ *
+ * @param  object_instance - object-instance number of the object
+ *
+ * @return  event-state property value
+ */
+unsigned Binary_Input_Event_State(uint32_t object_instance)
+{
+    unsigned state = EVENT_STATE_NORMAL;
```

Comment:
might get an args unused type warning for object_instance if we don't go into the #if

Reply:
Aye - changed it.